package.json

```diff
@@ -34,7 +34,6 @@
     "react-dom": "18.2.0"
   },
   "volta": {
-    "node": "18.14.0",
-    "npm": "9.4.2"
+    "node": "19.7.0"
```

Comment:
Vi er sikre på at vi vil opp til Node 19? `18.14.2` er vel den som er lts nå.

Reply:
Var mest for å teste om det hjalp, men det gjorde det ikke. Node 19 er ikke LTS, men vi har vel ikke behov for å være på LTS siden vi aktivt utvikler designsystemet.

Reply:
Dropper oppgradering av node og npm nå uansett, kan ta det i en egen PR